app/main.py

```diff
@@ -1,8 +1,61 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        if not 1 <= comfort_class <= 7:
+            raise "Comfort class must be between 1 and 7."
+        if not 1 <= clean_mark <= 10:
+            raise "Clean mark must be between 1 and 10."
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int, average_rating: float,
+                 count_of_ratings: int):
+        if not (1.0 <= distance_from_city_center <= 10.0):
+            raise "Distance from city center must be between 1.0 and 10.0."
+        if not 1 <= clean_power <= 10:
+            raise "Clean power must be between 1 and 10."
+        if not 1.0 <= average_rating <= 5.0:
+            raise "Average rating must be between 1.0 and 5.0."
```

Comment:
Error: Raising exceptions with string literals is incorrect. Use `raise ValueError('Average rating must be between 1.0 and 5.0.')` instead.